tt_metal/llrt/llrt.cpp

```diff
@@ -304,6 +338,29 @@ static bool check_if_riscs_on_specified_core_done(chip_id_t chip_id, const CoreC
     }
 }
 
+static bool check_if_erisc_on_specified_core_done(chip_id_t chip_id, const CoreCoord &core, int run_state) {
```

Comment:
same with this, do we need a dedicated routine?

Reply:
Because the l1 maps are slightly different w.r.t where the launch message is sitting in L1, the two functions use their specific defines.
It can be combined with Brisc version and named a generic risc_on_specified_core instead of brisc/erisc_on_specified_core